documentation/documentation/docs/projectform/project_upload_form.md

```diff
@@ -0,0 +1,34 @@
+# Project upload guide
+A teacher can upload a new project of their course by going to the url ```projects/create``` where they will need to fill out data corresponding to the project they wants to upload.
+![Projectform](/img/project_form_1.png)
+
+The first 3 fields are rather straightforward, you just need to fill in the title you want for the project, the description and select the course you want the project to be for. The course selection is in a dropdown menu where you can see all the courses that you are the teacher of.
```

Comment:
No need to say it's straightforward, scratch that

Reply:
https://github.com/SELab-2/UGent-3/pull/250/commits/a175039634b28c72360c862b6f82fb2f88d1afe0